src/App.js

```diff
@@ -1,8 +1,18 @@
 import "./App.css";
 import { Homepage } from "./pages/Homepage.js";
+import {ProductListing} from "./pages/Product-Listing.js";
+import {Routes,Route} from "react-router-dom";
+
 
 function App() {
-  return <Homepage />;
+  return (
+    
+  <Routes>
+    <Route path="/" element={<Homepage />} />
+    <Route path="/products" element={<ProductListing/>} />
```

Comment:
React router v6 doesn't need `/` in the path. You can omit that

Reply:
Thankyou Sharath, I'll make all the code refactoring suggested